### PR TITLE
Adjusted

### PR DESCRIPTION
**Pull request by BrunoSonego**
Adjusted buttons sizes and positions to dont overlap

### CODE REVIEW

File: style/Plain/ui_800x480.json
Comment (eliasbakken) [inline style/Plain/ui_800x480.json]: Is this to hide the button? If so, I think it would be better to do it programatically based on the number of extruders.

Comment (goeland86) [reply style/Plain/ui_800x480.json]: I agree. Otherwise you keep the one pixel that may trigger an unwanted event without realizing it.

Comment (BrunoSonego) [reply style/Plain/ui_800x480.json]: Yes it was to hide it sure is better to hide it in extruder number

Comment (eliasbakken) [reply style/Plain/ui_800x480.json]: Could you change this to the same size? I know it's annoying if you only have one extruder, but it can't be accepted as it is.

Comment (BrunoSonego) [reply style/Plain/ui_800x480.json]: ok width set to 50 same as others

**Review by goeland86 — APPROVED** [review]
Waiting for @eliasbakken to merge this as I don't have an 800x480 screen to test it with

**Comment by eliasbakken** [comment]
I'll take a look at this on an actual screen tomorrow before merging.

**Pull request merged**
